hack/run-test-e2e-serial.sh

```diff
@@ -12,5 +26,37 @@ if [ -n "${E2E_SERIAL_FOCUS}" ]; then
 	FOCUS="-ginkgo.focus=${E2E_SERIAL_FOCUS}"
 fi
 
-echo "Running Serial, disruptive E2E Tests"
-${BIN_DIR}/e2e-serial.test ${NO_COLOR} --ginkgo.v --ginkgo.failFast --ginkgo.reportFile=/tmp/artifacts/e2e-serial --test.parallel=1 ${FOCUS}
+function setup() {
+	if [[ x"${SETUP}" != "xtrue" ]]; then
+		return 0
+	fi
+	echo "Running NRO install test suite"
+	${BIN_DIR}/e2e-install.test ${NO_COLOR} --ginkgo.v --ginkgo.failFast --ginkgo.reportFile=/tmp/artifacts/e2e-serial-install --ginkgo.focus='\[Install\] continuousIntegration'
```

Comment:
This is not enough. it's only the NROP CR installation.
We also need the ```NUMAResourcesScheduler``` CR from here:
https://github.com/openshift-kni/numaresources-operator/blob/main/hack/run-test-e2e.sh#L8-L26

Reply:
good point, addressing